addons/nodot/interaction/Portal3D.gd

```diff
@@ -0,0 +1,67 @@
+#Write by: Z3R0_GT 
+#Time: 04/19/23
+#Last update: 04/19/23
+
+##Portal3D node
```

Comment:
Please see this file as an example of how to start the file: https://github.com/NodotProject/nodot/blob/main/addons/nodot/detection/ViewCone3D.gd